Resources/Private/Fusion/Components/Atom/Section/Section.fusion

```diff
@@ -32,8 +32,9 @@ prototype(DIU.Grid:Components.Atom.Section) < prototype(Neos.Fusion:Component) {
     id = ''
 
     renderer = afx`
-        <section class={'atom-section ' + props.classNames + ' ' + props.offset}>
-            <div class={'container' + props.fluidLayout}>
+        <section class={'atom-section ' + props.classNames + ' ' + props.offset} id={props.anchorTag}>
+            <div  class={'container' + props.fluidLayout}>
+                <div @if.has={props.backgroundImage} class="a-s__background-image" style={'background-image: url("' + props.backgroundImage + '");'}></div>
```

Comment:
Muss das ein separates div Element sein, oder kann dies auch mit dem div container gelöst werden?

Reply:
You're right. see last commit